content/api/events.apib

```diff
@@ -0,0 +1,179 @@
+
+<MessageEvents></MessageEvents>
+
+___
+
+__The SparkPost Events API uses MaxMind software [MaxMind License](https://www.maxmind.com/download/geoip/database/LICENSE.txt)__
+
+___
+
+### Search for Message Events [GET /v1/events/message{?recipients,from,to,timezone,cursor,per_page,events,subaccounts,campaign_ids,template_ids,message_ids,transmission_ids,from_addresses,bounce_classes,reason,subject,sending_domains,sending_ips,ip_pools,ab_test_ids,ab_test_version}]
+
+Returns a list of message events that matched the filtered search. The response is sorted by descending timestamp.
+
++ Parameters
+    + from (string, optional) - Datetime in format of YYYY-MM-DDTHH:MM.
+        + Default: `1 minute ago`
+    + to (string, optional) - Datetime in format of YYYY-MM-DDTHH:MM.
+        + Default: `24 hours ago`
+    + timezone (string, optional) - Standard timezone identification string.
+        + Default: `UTC`
+    + cursor (string, optional) - Results cursor for pagination. Used in conjunction with `per_page` parameter.
+        + Default: `initial`
+    + per_page (number, optional) - Maximum number of results to return per page. Must be between 1 and 10,000.
+        + Default: `1000`
+    + events (string, optional) - Comma delimited list of event types to search. Defaults to all event types.
+    + subaccounts (string, optional) - Comma delimited list of subaccount IDs to search.
+    + recipients (string, optional, `recipient@example.com`) ... Comma delimited list of recipients to search. Supports wildcarding in both the localpart and domain part of the email address. For example, `*@gmail.com` and `sparky@yahoo.*` are both valid values.
+    + campaign_ids (string, optional) - Comma delimited list of campaign IDs to search.
+    + template_ids (string, optional, `my-template`) ... Comma delimited list of template IDs to search.
+    + message_ids (string, optional) - Comma delimited list of message IDs to search.
+    + transmission_ids (string, optional) - Comma delimited list of transmission IDs to search.
+    + from_addresses (string, optional) - Comma delimited list of friendly from addresses to search.
+    + bounce_classes (number, optional) - Comma delimited list of bounce classification codes to search. See [Bounce Classification Codes](https://www.sparkpost.com/docs/deliverability/bounce-classification-codes/).
+    + reason (string, optional) - Bounce/failure/rejection reason that will be matched using a wildcard (e.g., %reason%).
+    + subject (string, optional) - Subject line from the email header that will be matched using a wildcard (e.g., %avocados%).
+    + sending_domains (string, optional) - Comma delimited list of Sending Domains to search.
+    + sending_ips (string, optional) - Comma delimited list of IP addresses to search.
+    + ip_pools (string, optional) - Comma delimited list of IP pool IDs to search.
+    + ab_test_ids (string, optional) - Comma delimited list of A/B test IDs to search.
```

Comment:
if you specify version, you MUST specify an ID. wildcarding is that you didn't put in a version